edb/server/server.py

```diff
@@ -1564,17 +1580,29 @@ def create_task(self, coro, *, interruptable):
         # randomly in the middle when the event loop stops; while tasks with
         # interruptable=False are always awaited before the server stops, so
         # that e.g. all finally blocks get a chance to execute in those tasks.
-        if self._accept_new_tasks:
+        if self._task_group is None:
+            # Silence the "coroutine not awaited" warning by creating the task
+            # and cancelling it immediately.
+            detail = getattr(coro.cr_code, 'co_name', 'unknown')
+            origin = getattr(coro, 'cr_origin', None)
+            if origin:
+                detail += "\n" + "\n".join(
+                    f"    {path}, line {lineno}, in {method}"
+                    for path, lineno, method in origin
+                )
+            else:
+                detail += (
+                    "; set sys.set_coroutine_origin_tracking_depth() to debug"
+                )
+            logger.debug("Task is not started and ignored: %s", detail)
+            task = self.__loop.create_task(coro)
+            task.cancel()
```

Comment:
If we are to keep this code, please move it to a function. Because when reading this code 60% of `craete_task` being cancellation is a distraction.